src/View/View.php

```diff
@@ -1316,7 +1317,7 @@ protected function _checkFilePath(string $file, string $path): string
      *
      * @param string $name The name you want to plugin split.
      * @param bool $fallback If true uses the plugin set in the current Request when parsed plugin is not loaded
-     * @return array Array with 2 indexes. 0 => plugin name, 1 => filename
+     * @return string[] Array with 2 indexes. 0 => plugin name, 1 => filename
```

Comment:
Here too plugin name (value at 0 index) can be `null` hence `string[]` is incorrect.